FSJ_django20_project/FSJ/views.py

```diff
@@ -46,11 +46,20 @@ def profile(request):
             profile_form = AdjudicatorRestrictedForm(instance=FSJ_user)
         elif isinstance(FSJ_user, Coordinator):
             profile_form = CoordinatorRestrictedForm(instance=FSJ_user)        
-            
+
     context = get_standard_context(FSJ_user)
     template = loader.get_template("FSJ/profile.html")
     context["form"] = profile_form
     url = "/profile/"
     context["url"] = url
     return HttpResponse(template.render(context, request))
 
+@login_required
+@user_passes_test(is_FSJ_user)
```

Comment:
The implementation (comment included) has code paths for Coordinators and Students but not for Adjudicators. Is this @user_passes_test the correct function to call (i.e., are adjudicators supposed to access this page in some way?

Reply:
I've changed this to is_coordinator instead, when we implement a student awards page, this will be changed